include/mbedtls/config.h

```diff
@@ -784,34 +784,6 @@
  */
 //#define MBEDTLS_ECP_RESTARTABLE
 
-/**
```

Comment:
In the documentation of MBEDTLS_ECP_RESTARTABLE there is a mention to MBEDTLS_ECDH_LEGACY_CONTEXT which I think was wrong and should be removed.